### PR TITLE
Development2

### PR DESCRIPTION
**Pull request by mosesogwo**
This is our submission for the mint.com signup page project.
Awaiting your review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    </head>
+    <body>
+        <div class="top">
+            <div class="top-text">
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <div class="signin-btn">
+                <a href="#">
+                    <button type="button">Sign In</button>
```
Comment (mcrd25) [inline index.html:16]: A button cannot be a descendant of an `a` tag.

Use either a` button` tag or an `a` tag but not one inside another. 

If you are looking for the hover effect for the button you can do so, for now, with `cursor: pointer` CSS property value pair for your button or you can style the anchor tag to be like a button.

Hope this helps. 😄

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    text-align: center;
+}
+
+h3 {
+    display: inline;
+}
+
+h3 a {
+    color: black;
+    text-decoration: none;
```
Comment (mcrd25) [inline styles/style.css]: I noticed you have `text-decoration: none` a few times maybe you can put this for all anchor tags and the one case where you want it (top right corner or webpage) then you'd add it using `text-decoration` property.

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# One more review is needed and you can 🚀 

What I liked:
- you did all that was required
- you kept it simple
- you added your names to the README
- You set up stickler successfully
- Address review comments
Suggested (not required) changes
- You can further improve by making your buttons clickable like original by using `cursor: pointer` or similar (suggested not required)
- Maybe you can decrease the width of the form a tad bit
- You can increase the height of input as well
- add a link to your live version of your clone in README (you can udate it also when merged to master to production) I used [githack](https://raw.githack.com/) and obtained this link: https://raw.githack.com/mosesogwo/Mint-SIgnup/development2/index.html. 
You can do same by visiting and using your index.html blob [link](https://github.com/mosesogwo/Mint-SIgnup/blob/development2/index.html) from development2 branch.

**In general, your project is great and would have been approved had it not been for the button inside an `a` tag situation! so keep up the great work! Make changes and submit another review!** 👍 💯 

***If you have any questions, let me know here or on slack and I will be glad to clarify with you.***

**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

File: index.html (hunk 1 above)
Comment (mosesogwo) [reply index.html:16]: Thanks Maya for your review.
The a tag has been removed.

File: styles/style.css (hunk 2 above)
Comment (mosesogwo) [reply styles/style.css]: This suggestion has been implemented.
Thanks.

**Review by mcrd25 — APPROVED** [review]
# Well done! You may now merge to master! 🚀 ㊗️ 🎉 

***If you have any questions, concerning this project moving forward that you'd like to improve, feel free to contact me anytime!***

**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.form-footer p {
+    font-size: 13px;
+    margin: 0;
+    padding: 15px;
+    text-align: center;
+}
+
+a {
```
Comment (mcrd25) [inline styles/style.css:117]: Maybe, for structure, you can put this on closer to the beginning of file way you implement general styles.

**Pull request closed without merging**